Resources/Prototypes/_RMC14/Entities/flora.yml

```
@@ -0,0 +1,267 @@
+  - type: Climbable
+
+- type: entity
+  parent: RMCBaseRockIndistructable
+  id: RMCBaseRockDestructible
+  components:
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+      - trigger:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)